app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: str) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
+
+    @classmethod
+    def get_person(cls, name: str) -> "Person":
+        return cls.people.get(name)
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    for person in people:
+        Person(person["name"], person["age"])
+    for person in people:
+        new_person = Person.get_person(person["name"])
+        if person.get("wife") is not None:
```

Comment:
no need to write `is not None` here, condition won't be True if get method will return None